<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>log_pars</class>
 <widget class="QMainWindow" name="log_pars">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0" colspan="2">
         <widget class="QTableView" name="tableView"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Mesačné štatistiky:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="0">
             <widget class="QPushButton" name="FIND_FILE">
              <property name="text">
               <string>Nastav priečinok</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>resources/dir2.jpg</normaloff>resources/dir2.jpg</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QComboBox" name="cb_open_file">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Logovacie správy</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" rowspan="3">
             <widget class="QLabel" name="label_log_msg_path">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QCheckBox" name="checkBox_month_stat">
              <property name="text">
               <string>Dopočítať aj month stat</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PBtn_find_stat_path">
              <property name="maximumSize">
               <size>
                <width>130</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="palette">
               <palette>
                <active>
                 <colorrole role="Button">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>255</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="Button">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>255</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="Button">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="text">
               <string>Nastav cestu k dátam...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="Line" name="line_2">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="4">
           <widget class="Line" name="line_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_export_excel">
              <property name="text">
               <string>Exportuj statistiku do excelu</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCalendarWidget" name="calendarWidget">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>160</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>160</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Manuálne dekódovanie správy:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="btn_clear_table">
              <property name="text">
               <string>Clear table</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="plainText">
             <string>Sem vlož logovú správu</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView_manual"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
